fluent_pages/pagetypes/fluentpage/models.py

```diff
@@ -3,11 +3,14 @@
 from fluent_pages.models import PageLayout
 from fluent_pages.integration.fluent_contents.models import FluentContentsPage
 
+from parler.fields import TranslatedField
+from parler.models import TranslatableModel, TranslatedFieldsModel
+
 
 # This all exists for backwards compatibility
 # The new v0.9 method is using fluent_pages.integration.fluent_contents, instead of directly inheriting this app.
 # In that way, the pagetypes are stand-alone apps again.
-class AbstractFluentPage(FluentContentsPage):
+class AbstractFluentPage(FluentContentsPage, TranslatableModel):
```

Comment:
`FluentContentsPage` already inherits from `TranslatableModel`, so this change seems unnecessary.
(FluentContentsPage -> HtmlPage -> Page -> UrlNode -> PolymorphicMPTTModel + TranslatableModel)